pages/operations/managing-applications/update-application.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Update an Application
+description: How to change an application's version.
+---
+
+Updates to Plural applications are applied **automatically** based on a per-application setting. This setting supports telling Plural to only upgrade for Stable or Warm releases, as well as disabling automatic upgrades altogether.
+
+You can change this setting by clicking the gear in the top-right corner of the application's detail page in the Plural app. You can find your installed applications at [https://app.plural.sh/installed](https://app.plural.sh/installed).
+
+![](</assets/operations/update-application.png>)
+
+## Rollback to a previous version
+
+This requires having the [Plural Console installed](../../getting-started/admin-console.md). We'll need this to create an upgrade
+policy that tells Plural to not deploy an upgrade to your application, which is normally performed automatically unless disabled using the setting in our app. You could just disable upgrades altogether in
```

Comment:
Is the idea here that there are scenarios where users will want to avoid a single upgrade for a period of time? I could see disabling upgrades if they want to retain a specific version, but is this intended to roll back in the case of a broken upgrade?

Reply:
Both, essentially. But more of the latter, to rollback for broken upgrades.